kconfiglib.py

```diff
@@ -420,6 +420,18 @@
 
 'source' and 'osource' are analogous to 'include' and '-include' in Make.
 
+Listing 'source'
+----------------
+
+'lsource' accepts a list of patterns for sourcing. For example,
+
+  lsource "foo bar"
+
+is equivalent to
+
+  source "foo"
+  source "bar"
+
```

Comment:
Could add something like

```
When combined with the Kconfig preprocessor, this can be used to include a list
of files from an environment variable, like

  lsource "$(ENV_VAR)"

A preprocessor function could be used as well.
```